src/quo/components/list_items/user.cljs

```diff
@@ -17,26 +17,28 @@
    :flex-direction     :row
    :align-items        :center})
 
-(defn action-icon
-  [{:keys [type on-press on-check disabled? checked?]} customization-color theme]
-  [rn/touchable-opacity
-   {:on-press on-press}
-   (case type
-     :options
-     [icons/icon :i/options
-      {:size  20
-       :color (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)}]
-     :checkbox
-     [selectors/view
-      {:type                :checkbox
-       :checked?            checked?
-       :customization-color customization-color
-       :accessibility-label :user-list-toggle-check
-       :disabled?           disabled?
-       :on-change           (when on-check on-check)}]
-     :close
-     [text/text "not implemented"]
-     [rn/view])])
+(defn accessory-item
+  [{:keys [type on-press on-check disabled? checked? child]} customization-color theme]
+  (if (= type :custom)
+    child
```

Comment:
please comment if you have better ideas :)

Reply:
An alternative could be having a :button type and pass the label/on-press, but we should also address this with the design team since they went beyond the design system here it seems.